code/FileAttachmentFieldTrack.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/**
+ * Track files as they're uploaded and remove when they've been saved.
+ *
+ * @package  unclecheese/silverstripe-dropzone
+ */
+class FileAttachmentFieldTrack extends DataObject {
+    private static $db = array(
+        'SessionID' => 'Varchar(255)',
+        'ControllerClass' => 'Varchar(60)',
+        'RecordID' => 'Int',
+        'RecordClass' => 'Varchar(60)',
+    );
```

Comment:
The idea here is that if you decide to never setup the cron or modify the cleanup task due to inexperience or lazyness, you'll know exactly what tracking items relate to what. This is to catch cases where someone has built a custom Form, and hasn't untracked a value properly and needs to fix the data later.